Jenkinsfile

```diff
@@ -42,41 +38,33 @@ node {
       defraUtils.replaceInFile(containerSrcFolder, localSrcFolder, lcovFile)
     }
     stage('SonarQube analysis') {
-      defraUtils.analyseCode(sonarQubeEnv, sonarScanner, ['sonar.projectKey' : repoName, 'sonar.sources' : '.'])
+      defraUtils.analyseCode(sonarQubeEnv, sonarScanner, ['sonar.projectKey' : serviceName, 'sonar.sources' : '.'])
     }
     stage("Code quality gate") {
       defraUtils.waitForQualityGateResult(timeoutInMinutes)
     }
     stage('Push container image') {
-      defraUtils.buildAndPushContainerImage(regCredsId, registry, imageName, containerTag)
+      defraUtils.buildAndPushContainerImage(DOCKER_REGISTRY_CREDENTIALS_ID, DOCKER_REGISTRY, serviceName, containerTag)
     }
     if (pr != '') {
       stage('Verify version incremented') {
         defraUtils.verifyPackageJsonVersionIncremented()
       }
       stage('Helm install') {
         withCredentials([
-          string(credentialsId: 'sqsQueueEndpoint', variable: 'sqsQueueEndpoint'),
-          string(credentialsId: 'calculationQueueUrlPR', variable: 'calculationQueueUrl'),
-          string(credentialsId: 'calculationQueueAccessKeyIdSend', variable: 'calculationQueueAccessKeyId'),
-          string(credentialsId: 'calculationQueueSecretAccessKeySend', variable: 'calculationQueueSecretAccessKey'),
-          string(credentialsId: 'scheduleQueueUrlPR', variable: 'scheduleQueueUrl'),
-          string(credentialsId: 'scheduleQueueAccessKeyIdSend', variable: 'scheduleQueueAccessKeyId'),
-          string(credentialsId: 'scheduleQueueSecretAccessKeySend', variable: 'scheduleQueueSecretAccessKey'),
-          string(credentialsId: 'postgresExternalNameClaimsPR', variable: 'postgresExternalName'),
-          usernamePassword(credentialsId: 'postgresClaimsPR', usernameVariable: 'postgresUsername', passwordVariable: 'postgresPassword'),
-          ]) {
+          string(credentialsId: 'sqs-queue-endpoint', variable: 'sqsQueueEndpoint'),
+          string(credentialsId: 'calculation-queue-url-pr', variable: 'calculationQueueUrl'),
+          string(credentialsId: 'schedule-queue-url-pr', variable: 'scheduleQueueUrl'),
+          string(credentialsId: 'postgres-external-name-claims-pr', variable: 'postgresExternalName'),
```

Comment:
I've created a `postgres-external-name-pr` credential that could be used for all PRs as they are pointing to the same server, saves a few credentials we'll have to change